backend/database/redis_db.py

```diff
@@ -137,6 +137,19 @@ def get_plugin_reviews(plugin_id: str) -> dict:
         return {}
     return eval(reviews)
 
+def get_plugins_reviews(plugin_ids: list) -> dict:
+    if not plugin_ids:
+        return {}
+
+    keys = [f'plugins:{plugin_id}:reviews' for plugin_id in plugin_ids]
+    reviews = r.mget(keys)
+    if reviews is None:
+        return {}
+    return {
+        plugin_id: eval(review) if review else {}
+        for plugin_id, review in zip(plugin_ids, reviews)
```

Comment:
what happen if a plugin does not have any review, does zip(x,y) still work as expected?

Reply:
i tested and it worked
keys: [plugin1, plugin2, plugin3]
plugin2 dont have review -> mget will return: [review1, None, review2]
-> this code above will return: {plugin1: review1, plugin2: {}, plugin3: review3}